csharp/extractor/Semmle.Extraction.CSharp/Entities/Property.cs

```diff
@@ -74,7 +74,7 @@ public override void Populate(TextWriter trapFile)
                 WriteLocationsToTrap(trapFile.property_location, this, Locations);
             }
 
-            if (IsSourceDeclaration && Symbol.FromSource())
+            if (IsSourceDeclaration && Symbol.FromSource() && !Context.OnlyScaffold)
```

Comment:
I think line 68 should be guarded as well.

Reply:
I agree. I will take a bit more structured approach to not extract TypeMentions when scaffolding (as they use `*` IDs as well).